src/reducers/index.js

```diff
@@ -8,11 +8,16 @@ import item from './item';
 
 const persistConfig = {
   key: 'auth',
-  storage
+  storage,
+};
+
+const persistItemConfig = {
```

Comment:
Happy with this to go in for now but we'll need to figure out how to add this in with one config object.